spec/tbd_tests_spec.rb

```diff
@@ -1471,7 +1481,9 @@
       if surface.has_key?(:ratio)
         ratio  = format "%3.1f", surface[:ratio]
         name   = id.rjust(15, " ")
-        puts "#{name} RSi derated by #{ratio}%"
+        #puts "#{name} RSi derated by #{ratio}%"
+      else
+        expect(surface[:boundary].downcase).to_not eq("outdoors")
```

Comment:
This small test should ensure that seb surface constructions not subjected to some degree of thermal derating should not be facing Outdoors. This may fail however if subjected to thermal bridges PSI values of 0.00 (i.e. a failsafe mode). May warrant more attention in future revisions.